packages/remote-config/android/src/main/java/io/capawesome/capacitorjs/plugins/firebase/remoteconfig/FirebaseRemoteConfigPlugin.java

```diff
@@ -76,9 +79,12 @@ public void error(String message) {
     @PluginMethod
     public void fetchConfig(PluginCall call) {
         try {
-            int minimumFetchIntervalInSeconds = call.getInt("minimumFetchIntervalInSeconds", 43200);
+            Integer minimumFetchIntervalInSeconds = call.getInt("minimumFetchIntervalInSeconds", DEFAULT_MINIMUM_FETCH_INTERVAL_IN_SECONDS);
+            if (minimumFetchIntervalInSeconds == null) {
+                minimumFetchIntervalInSeconds = DEFAULT_MINIMUM_FETCH_INTERVAL_IN_SECONDS;
+            }
```

Comment:
This code should not be necessary.

Reply:
@robingenz Yes, I've deleted it for now.https://github.com/capawesome-team/capacitor-firebase/pull/694/commits/775f94d2e94efc5711bd16d44b27057ecf9c2988

There should be no issues with the program's functionality, but this warning appears.
> Method invocation 'longValue' may produce 'NullPointerException'